tools/automator/automator.sh

```diff
@@ -257,7 +257,15 @@ merge() {
   git -c "user.name=$user" -c "user.email=$email" merge --no-ff -m "$title" --log upstream/"$merge_branch"
   local code=$?
   if [ "$code" -ne 0 ]; then
-    print_error "$(git status)" 1
+    export GITHUB_TOKEN="$token"
+    local issue_exists
+    issue_exists=$(gh issue list -S "Automatic merge of $merge_branch into $branch failed." -R "istio/isio" | wc -l)
```

Comment:
will this list only open issues? I.e.:
*  if it fails once
* opens an issue
* someone fixes it and it continues
* another merge failure occurs -- what happens here?
If I understand correctly from the gh docs, then this will find the original issue and not open a new one?

Reply:
`list` shows open issues only; my assumption is that these issues will be closed as soon as they have been resolved; the worst case is that we won't see new issues while there's one open.

Reply:
Thanks. Didn't try testing it, but the docs made it look like they would show anything.

Reply:
I think it's safe to trust that the issues will be closed once fixed. Otherwise it's hard to differentiate.